mcs/class/System/Test/System.Diagnostics/SourceSwitchTest.cs

```diff
@@ -53,7 +53,7 @@ public class SourceSwitchTest
 		public void ConstructorNullName ()
 		{
 			SourceSwitch s = new SourceSwitch (null);
-			AssertHelper.IsEmpty (s.DisplayName);
+			Assert.AreEqual (s.DisplayName.Length, 0);
```

Comment:
So It looks like this changed from when I started my BCL work. This commit is not in 4.8 (where we were tracking in master and I started this):

https://github.com/mono/mono/commit/863dc73d1984175cb52734e1cb755c9d57de53bd

And my resolve kept my changes. These changes now look unnecessary (at least in master, and we can cherry pick that into 4.8 if necessary).